src/content/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1220.mdx

```diff
@@ -4,8 +4,13 @@ releaseDate: '2022-10-07'
 version: '1220'
 ---
 
+<Callout variant="caution" title="About this release">
```

Comment:
@coreyarnold Is it okay to keep it in the caution callout as a preface to the bullet points?